.env.sample

```diff
@@ -0,0 +1,67 @@
+#################################################
+# Required values
+#################################################
+
+# Database connection string (required)
+# In development it's possible to set this to a SQLite database,
+# but we recommend running a MySQL instance locally
+# for parity with production.
+DB_CONNECTION_URI=mysql://some-mysql-connection-string
+
+# Staff auth redirect url (required)
+# This is set in the environment as it needs to be defined at
+# start up time so can't be computed based on the request.
+MS_REDIRECT_URL=http://localhost:3000/user/staff/auth/openid/return
+
+# Origin URLs for test and production environments
+# Required if using the bundled deploy script in ./bin/deploy
+# Ask another developer for these values
+# PROD_URL=
+# TEST_URL=
+
+#################################################
+# Optional local overrides
+#################################################
+
+# Content API url (optional)
+# If you are making changes to the CMS locally
+# you may want to point your app to a local API url.
+# By default this value is taken from the secrets.
+# CONTENT_API_URL=
+
+# Past grants API url (optional)
+# If you are making changes to the grants servoce locally
+# you may want to point your app to a local API url.
+# By default this value is taken from the secrets.
+# PAST_GRANTS_API_URI=
+
+# Salesforce API config (optional)
+# You may want to override these values if testing against
+# a different Salesforce instance.
+# By default these values are taken from the secrets.
+# SALESFORCE_API_URL=
+# SALESFORCE_CONSUMER_KEY=
+# SALESFORCE_CONSUMER_SECRET=
+# SALESFORCE_USERNAME=
+# SALESFORCE_PASSWORD=
+# SALESFORCE_TOKEN=
+# SALESFORCE_INSTANCE_ID=
+
+# ClamAV settings
+# If you are running clamd locally you may need
+# to set custom paths depending on your configuration
+# CLAMDSCAN_SOCKET=
+# CLAMDSCAN_PATH=
+# CLAMDSCAN_CONFIG_FILE=
+
+# Watch nunjucks templates for changes (default: false)
+# This is disabled by default as it can cause issues with
+# file watchers in CI environments, but is useful locally
+# WATCH_TEMPLATES=true
+
+# Can be enabled in development to show a notification
+# when nodemon restarts (default: false)
+# SHOW_RESTART_NOTIFICATION=true
+
+# Set a custom local port (default: 3000)
+# PORT=
```

Comment:
Might need this adding:
```

# Material supplier email address
# A custom email address to send material orders (free plaques etc) to.
# Defaults to the value in secrets
# MATERIAL_SUPPLIER=
```

Reply:
There might be a few others but I'm on my personal laptop with an outdated `.env` file. Think there's one for sending expiry emails though, `EMAIL_EXPIRY_TEST_ADDRESS`

Reply:
Interesting about the `MATERIAL_SUPPLIER` value. We have this in the code https://github.com/biglotteryfund/blf-alpha/blob/master/controllers/materials/index.js#L116-L118

Looking at that, we always you the recipients email outside of production (to avoid ever mistakenly sending to the supplier) so that environment variable is probably redundant!

Good shout about `EMAIL_EXPIRY_TEST_ADDRESS` though.

Reply:
Oh ha, that's funny – yeah, ignore that one then!